homeassistant/helpers/config_validation.py

```diff
@@ -233,8 +233,14 @@ def template(value):
     """Validate a jinja2 template."""
     if value is None:
         raise vol.Invalid('template value is None')
-    if isinstance(value, (list, dict)):
-        raise vol.Invalid('template value should be a string')
+    if isinstance(value, list):
```

Comment:
And also on notify data it could have more deep like telegram. I think we do that right and once. I will don't update that after new things will come. The valid config represent what is going with core.